src/largest-number-in-sequence.js

```diff
@@ -0,0 +1,16 @@
+// const numberArray = [-1, 4, 2, 4, 6, 3, 1, 6, 7, 5, 300, 22, 100];
+function findLargestNumber(numberArray) {
```

Comment:
`findMax` but `findLargestNumber` will be fine. also numberArray can be named `numbers`